pkg/scheduler/apis/config/v1beta3/zz_generated.conversion.go

```diff
@@ -751,6 +751,7 @@ func Convert_v1beta3_Plugins_To_config_Plugins(in *v1beta3.Plugins, out *config.
 }
 
 func autoConvert_config_Plugins_To_v1beta3_Plugins(in *config.Plugins, out *v1beta3.Plugins, s conversion.Scope) error {
+	// WARNING: in.Enqueue requires manual conversion: does not exist in peer-type
```

Comment:
flagging this to make sure it was intentional this field was only added to the v1 type. The release note should be clear about what field is added and to which versions, and/or link to docs for the feature config